docs/getting-started/install.mdx

```diff
@@ -0,0 +1,63 @@
+# Install
+
+Welcome to Widgetbook! This guide will help you get started with integrating Widgetbook into your Flutter projects and streamline your widget management process.
```

Comment:
`Widget` instead of `widget`

Reply:
mm, well, I guess when we refer to widget as a general word it should be lowercase but when we refer to the class it should be Widget, what do oyu think?